src/test/resources/__files/prisoner-details/GET_prisoner_details_success_any_number.json

```diff
@@ -117,7 +117,7 @@
   "identifiers": [
     {
       "type": "PNC",
-      "value": "1231/XX/121",
+      "value": "20120052494O",
```

Comment:
Pretty sure this is still invalid - https://www.criminaljusticehub.org.uk/jargon-buster/police-national-computer-id-pnc-id/ says O is not a valid check letter. I think it should be a Q

Reply:
Yup.